docs/reference/specifications/providers.md

```diff
@@ -280,6 +280,7 @@ Below are the supported configuration parameters (note that not all apply to bot
 | cache                 | FLAGD_CACHE                    | enable cache of static flags                                           | String - `lru`, `disabled`   | lru                           | rpc                     |
 | maxCacheSize          | FLAGD_MAX_CACHE_SIZE           | max size of static flag cache                                          | int                          | 1000                          | rpc                     |
 | selector              | FLAGD_SOURCE_SELECTOR          | selects a single sync source to retrieve flags from only that source   | string                       | null                          | in-process              |
+| providerID            | FLAGD_PROVIDER_ID              | A unique identifier for flagd(grpc client) initiating the request.     | string                       | null                          | in-process              |
```

Comment:
Generally I like the idea, out of curiosity, could this be the domain of the provider?

Reply:
Nvm I see the purpose now to make it better visible on flagd to see from whom the request was done.